realm/realm-library/src/objectServer/java/io/realm/internal/network/AuthenticateResponse.java

```diff
@@ -132,14 +133,14 @@ private AuthenticateResponse(String serverResponse) {
             if (accessToken == null) {
                 message = "accessToken = null";
             } else {
-                message = String.format("Identity %s; Path %s", accessToken.identity(), accessToken.path());
+                message = String.format(Locale.ENGLISH, "Identity %s; Path %s", accessToken.identity(), accessToken.path());
```

Comment:
Will this still work if `accessToken.path()` return a non-english string? like CJK?